src/main/resources/logback.xml

```diff
@@ -0,0 +1,10 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
It'd be good to update the logback config to use json so that kibana can read them properly. You can copy the config from drt-v2 or drt-dashboard

Reply:
Using https://github.com/logstash/logstash-logback-encoder instead